src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -823,4 +826,12 @@ public static void onPostServerTick(BooleanSupplier haveTime)
     {
         MinecraftForge.EVENT_BUS.post(new TickEvent.ServerTickEvent(TickEvent.Phase.END, haveTime));
     }
+
+    public static WeightedRandomList<MobSpawnSettings.SpawnerData> getPotentialSpawns(LevelAccessor world, MobCategory type, BlockPos pos, WeightedRandomList<MobSpawnSettings.SpawnerData> oldList)
+    {
+        WorldEvent.PotentialSpawns event = new WorldEvent.PotentialSpawns(world, type, pos, oldList);
+        if (MinecraftForge.EVENT_BUS.post(event))
+            return WeightedRandomList.create ();
```

Comment:
Formatting: no space between method name and opening parenthesis.
```suggestion
            return WeightedRandomList.create();
```